docs/README.md

```diff
@@ -11,10 +11,26 @@ Pages](http://pages.github.com/).
 From the repository root run `npm run docs` and navigate your browser to
 `http://localhost:4000`. This will start an express base node server with
 webpack-dev middleware that will watch your file changes and recompile all the
-static assets needed to generate the site. 
+static assets needed to generate the site. In the console output you'll see that
+that we bind to two ports. The first port is the one you'll use to load the docs
```

Comment:
If that's the worst thing in this you can find, then I guess I can fix it :wink: